lib/rpc/modules/eth.ts

```diff
@@ -82,7 +82,7 @@ export class Eth {
    * as the second argument
    * @return {Promise}
    */
-  async getBlockByHash (params: string[] | boolean[], cb: (err: Error | null, val?: any) => void) {
+  async getBlockByHash (params: [string, number], cb: (err: Error | null, val?: any) => void) {
```

Comment:
According to the JSDoc description and underlying condition it should be `[string, boolean]` instead of `[string, number]`.